chapters/VK_EXT_debug_utils.txt

```diff
@@ -52,6 +59,16 @@ include::{generated}/api/protos/vkSetDebugUtilsObjectNameEXT.txt[]
     pname:pNameInfo->objectType must: not be ename:VK_OBJECT_TYPE_UNKNOWN
   * [[VUID-vkSetDebugUtilsObjectNameEXT-pNameInfo-02588]]
     pname:pNameInfo->objectHandle must: not be dlink:VK_NULL_HANDLE
+  * If the object referred to by pname:pNameInfo->objectHandle was
+    created, allocated or retrieved from a slink:VkDevice,
+    pname:device must: be the device the object was created,
+    allocated or retrieved from.
+  * If pname:pNameInfo->objectType is ename:VK_OBJECT_TYPE_INSTANCE or
+    ename:VK_OBJECT_TYPE_PHYSICAL_DEVICE, pname:device must: have been
+    created from the object referred to by pname:pNameInfo->objectHandle.
+  * If pname:pNameInfo->objectType is not ename:VK_OBJECT_TYPE_INSTANCE,
+    both of pname:device, and pname:pNameInfo->objectHandle must: have the
+    same slink:VkInstance as their ancestor.
```

Comment:
What if they have the same Instance as ancestor but not Physical Device, as it can happen for `VkDisplayKHR`?

VUs should not end with periods.

Reply:
I wonder if the last bullet point is even needed.  Or maybe

```
 * If pname:pNameInfo->objectType is an instance object (like ename:VK_OBJECT_TYPE_PHYSICAL_DEVICE)  then both of pname:device and pname:pNameInfo->objectHandle must have the same slink:VkInstance as their ancestor
```

Slightly diffierent, but I think satsifies @krOoze's concern?  Or do we not differentiate instance and device objects in that way?